Documentation/nvme-get-property.txt

```diff
@@ -19,8 +19,8 @@ Reads and shows the defined NVMe controller property for NVMe over Fabric.
 
 OPTIONS
 -------
--O::
---offset::
+-O <offset>::
+--offset=<offset>::
```

Comment:
Just split the two different changes into two patches. One which fixes the mode settings and one which fixes the offset changes. Thanks.